### PR TITLE
feat: readme in spanish

### PR DESCRIPTION
**Pull request by sesave**
Hello! 
New features:
- Readme in spanish

### CODE REVIEW

**Comment by willianrod** [comment]
@sesave Thanks for the contribution. Just one little thing, can you add a section adding into the main README linking to the readme in Spanish too? Maybe under the header and above the first section. I was thinking something like this:

```md
## The readme also available in:
- Español
- English
- Português
```

You don't need to add Portuguese for now, I put it there just to illustrate that the language should not be translated.

File: docs/README_ESP.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+- Agregar compatibilidad con Linux y MacOS para el panel de administración
+- Agregar compatibilidad con iOS para la aplicación ODeck
+
+### Vista previa del deck
+
+https://user-images.githubusercontent.com/6539258/154787245-872187a9-b05e-44c4-b162-42037ace352a.mp4
+
+### Vista previa del panel de administración
+
+https://user-images.githubusercontent.com/6539258/154787603-3cf7bcd0-f82c-4092-96b9-28546fe16568.mp4
```
Comment (willianrod) [inline docs/README_ESP.md:51]: I updated the readme to remove the media files that was in the `.media` directory, and use the new github  repository assets.

```suggestion
### Vista previa del deck

https://github.com/willianrod/ODeck/assets/6539258/9c58dbb1-38f6-4602-b211-aecbfc8e4dff

### Vista previa del panel de administración

https://github.com/willianrod/ODeck/assets/6539258/51710202-d665-4f8c-bd7c-17c470e8e29b
```

**Comment by sesave** [comment]
Sure! I'll do it tomorrow.

**Comment by sesave** [comment]
Hello! @willianrod The PR is done with the suggested changes.

I hope it's ok for you now.

**Pull request merged**